Sources/DataLoader/DataLoader.swift

```diff
@@ -1,4 +1,4 @@
-import NIO
+import NIOCore
 import NIOConcurrencyHelpers
```

Comment:
nit: Put `NIOCore` beneath `NIOConcurrencyHelpers` to make imports alphabetical and pass lint CI

Reply:
Fixed now.

Reply:
Ok, the last newline-related lint error only shows up with the latest version of `swiftformat`. Once I updated my local `swiftformat` version, was able to find and resolve that.

There appears to be a unit test failure in macOS that I can't reproduce locally. I might wait until #24 is merged, to see if that helps.

Reply:
Thank you!